### DISCUSSION TITLE
Black frames at beginning of cut clips

### DISCUSSION THREAD

**Discussion by Bobkovski** [mifi/lossless-cut#2132] [Support]
Hi,

I've been using Lossless Cut for about a year, to cut unwanted parts of the videos I shoot with my mirrorless camera. I just realized that the trimmed clips sometimes begin with a completely black frame, and a second frame with a black contour (a sort of irregular black frame). It creates a kind of quick fade in (from black). How can I prevent it?

I have done a search for similar discussions. It appears to have been a problem in 2019. The proposed solutions don't apply to my version of Lossless Cut (eg. there is no option called "nc" (normal cut), see https://github.com/mifi/lossless-cut/issues/177)

Thank you

**Comment by mifi** [comment]
hi

the setting "kc" / "nc" is now called "Keyframe cut mode" (it's a toggle on/off button), in the export dialog. also other suggestions like `avoid_negative_ts` setting and https://github.com/mifi/lossless-cut/blob/master/issues.md#the-exported-video-has-a-problem

**Reply by Bobkovski** [reply]
Hi Mikael,

I've tested all the proposed solutions except the "Experimental Flag" thing. The only one that works is to change the output format to matroska.

I'm worried about exporting all my trimmed videos in matroska. Apparently, audio isn't good with that format (https://www.dacast.com/blog/mkv-vs-mp4-for-video-streaming/)
My camera outputs .mp4 files. I have always chosen the .mov export format in LosslessCut, and it has always given .mp4 files (not .mov!). I was pretty satisfied with it (I thought it was better to go from a mp4 to another mp4).

The other option would be to test the "Experimental Flag", but I couldn't find what that is, while searching the web.

Or I'll manually remove the first 2 frames of each clip in Davinci Resolve when editing projects.

Any help would be appreciated!
Thank you

**Reply by mifi** [reply]
Unfortunately I don't have any silver bullet solution for you, other than the FAQ. Some files are a bit quirky. I don't think the Experimental Flag will help, but you can find it in the Export Confirm dialog, under "advanced".

**Comment by jauling** [comment]
I just ran into this issue as well. I have a h.264 video created from Insta360 footage, and I tried to split it into 2 pieces for social media posting. The second segment had a first black frame. I tried disabling all the "preserve" sliders and also enabled Experimental Flag, but still black frame remained.

I ended up just using the simple method of a direct ffmpeg copy operation, which worked fine.

```
ffmpeg -ss 00:00:00 -t 00:00:59 -i INPUT.mp4 -acodec copy -vcodec copy OUTPUT-seg1.mp4
ffmpeg -ss 00:00:59 -i INPUT.mp4 -acodec copy -vcodec copy OUTPUT-seg2.mp4
```

**Reply by mifi** [reply]
> I ended up just using the simple method of a direct ffmpeg copy operation, which worked fine.

that is exactly what losslesscut does under the hood, so i'm not sure why that worked, but not losslesscut. you can see which ffmpeg commands losslesscut generates in the "last ffmpeg commands" dialog in lsosslesscut
